common/constants.ts

```diff
@@ -220,3 +220,31 @@ export enum WAZUH_MODULES_ID{
 
 export const AUTHORIZED_AGENTS = 'authorized-agents';
 export const HEALTH_CHECK = 'health-check';
+
+// Health check
+export const HEALTH_CHECK_REDIRECTION_TIME = 300; //ms
+
+// Kibana settings
+// Default timeTilter set by the app
+export const WAZUH_KIBANA_SETTING_TIME_FILTER = {
+  from: "now-24h",
+  to: 'now'
+};
+export const KIBANA_SETTING_NAME_TIME_FILTER = 'timepicker:timeDefaults';
+
+// Default maxBuckets set by the app
+export const WAZUH_KIBANA_SETTING_MAX_BUCKETS = 200000;
+export const KIBANA_SETTING_NAME_MAX_BUCKETS = 'timelion:max_buckets';
+
+// Default metaFields Kibana setting set by the app
+export const WAZUH_KIBANA_SETTING_METAFIELDS = ['_source', '_index'];
+export const KIBANA_SETTING_NAME_METAFIELDS = 'metaFields';
+
+
+// Logger
+// log severity
+export type UILogSeverity = 'UI' | 'BUSSINESS' | 'CRITICAL';
```

Comment:
It would be nice to get severity options and level options from enum or const too
here is an example 
![image](https://user-images.githubusercontent.com/26828184/120698992-2e840d80-c486-11eb-9db6-27295776ddc2.png)